bin/porta

```diff
@@ -39,7 +47,7 @@ module Porta
     apicast_access_token: 'apicastsecret',
     apicast_registry_url: 'https://apicast-staging.proda.3sca.net/policies',
     porta_local_provider_api: 'http://provider-admin.3scale.localhost:3000',
-    docker_internal_host: 'host.docker.internal',
+    docker_internal_host: nil,
```

Comment:
`docker_internal_host` should be automatically detected based on platform and container runtime, so it is not necessary to be set by user

Reply:
We can remove the option altogether If we don't need it any more.

Reply:
I thought to leave it as a reference in case automatic mechanism does not work well in some environment and needs to be overridden. Or for testing purposes, in case one wants to check whther some particular IP or hostname will work instead of the auto-discovered one.

I shall put a comment to clarify.